### PR TITLE
fix: tidy rc and verbose

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
// -1 maybe cancel signal
// -2 if delete key failed
// add 'verbose' to the 'done' also
// check the 'err' of cmd.Run() also

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -101,13 +101,16 @@ func run() int {
...
 	cmd.Stderr = os.Stderr
-	err = cmd.Run()
 
+	err = cmd.Run()
 	code := cmd.ProcessState.ExitCode()
-	if code == 0 {
-		println(string(bk))
+	if code == 0 && err == nil {
+		if verbose {
+			println("[done] " + string(bk))
```
Comment (mattn) [inline main.go]:
```suggestion
			fmt.Fprintf(os.Stderr, "[done] %v\n", string(b))
```

File: main.go [hunk 2]
```diff
@@ -101,13 +101,16 @@ func run() int {
...
 	code := cmd.ProcessState.ExitCode()
-	if code == 0 {
-		println(string(bk))
+	if code == 0 && err == nil {
+		if verbose {
+			fmt.Fprintf(os.Stderr, "[done] %v\n", string(b))
+		}
 		if err = store.Delete(bk, &opt.WriteOptions{Sync: true}); err != nil {
 			fmt.Fprintf(os.Stderr, "%v: %v\n", os.Args[0], err)
+			return -2
```
Comment (mattn) [inline main.go:113]: exit code should be unsigned int

Comment (Shane-XB-Qian) [reply main.go:113]: pls check:
```
// ExitCode returns the exit code of the exited process, or -1
// if the process hasn't exited or was terminated by a signal.
```
tho i am not completely sure how to make it happen, :-)

File: main.go [hunk 3]
```diff
@@ -101,13 +101,16 @@ func run() int {
...
 	cmd.Stderr = os.Stderr
-	err = cmd.Run()
 
+	err = cmd.Run()
 	code := cmd.ProcessState.ExitCode()
-	if code == 0 {
-		println(string(bk))
+	if code == 0 && err == nil {
+		if verbose {
+			println("[done]", string(bk))
```
Comment (mattn) [inline main.go]: Do not use println. please use fmt.Fprint or fmt.Fprintln or fmt.Fprintf instead.

Comment (Shane-XB-Qian) [reply main.go]: ok
// i felt unfamiliar with this builtin 'println' also, but it was wrote there, so just re-used it.

**Comment by mattn** [comment]
Thank you

**Pull request merged**